client/views/view_units.cpp

```diff
@@ -33,59 +42,56 @@ units_view::units_view() : QWidget()
 
   // Configure the units table
   QStringList slist;
-  slist << _("Unit Type") << _("★ Upgradable") << _("⚒ In Progress")
-        << _("⚔ Active") << _("Shield Upkeep") << _("Food Upkeep")
+  slist << _("Unit Type") << _("★  Upgradable") << _("⚒  In Progress")
+        << _("⚔  Active") << _("Shield Upkeep") << _("Food Upkeep")
         << _("Gold Upkeep") << QLatin1String("");
   ui.units_label->setText(QString(_("Units:")));
   ui.units_widget->setColumnCount(slist.count());
   ui.units_widget->setHorizontalHeaderLabels(slist);
   ui.units_widget->setSortingEnabled(false);
+  ui.units_widget->setAlternatingRowColors(true);
   ui.upg_but->setText(_("Upgrade"));
+  ui.upg_but->setToolTip(_("Upgrade selected unit."));
+  ui.upg_but->setToolTipDuration(5000);
```

Comment:
The default duration is calculated automatically based on the length of the tooltip, why change it? The formula is `10000 + 40 * qMax(0, text.length()-100)`, would give 10000 in this case.